src/components/SearchInput.vue

```diff
@@ -60,7 +59,13 @@ export default defineComponent({
   },
   setup() {
     const schema = yup.object({
-      did: yup.string().trim().required().min(50).max(50),
+      did: yup
+        .string()
+        .matches(/^did:vda:testnet:.+$/)
```

Comment:
You should keep a note somewhere (issue?) of this 'testnet' as it's likely to need an update when using non-testnet DID.

Better option would be to use the veridaEnv variable to adapt the match.
(I assume the did:vda:testnet are only on Verida Testnet, right?